tools/gendocs/gendocs/gendocs.go

```diff
@@ -70,18 +70,21 @@ func GenDocs(cmd *cobra.Command, filename string, admin bool) error {
 	return nil
 }
 
-func extractExamples(cmd *cobra.Command, admin bool) Examples {
+func extractExamples(cmd *cobra.Command, admin, microshift bool) Examples {
 	objs := Examples{}
 	for _, c := range cmd.Commands() {
 		if len(c.Deprecated) > 0 {
 			continue
 		}
+		if microshift && microshiftCommands.Has(c.CommandPath()) {
```

Comment:
nit: It's a little odd to call it `microshiftCommands` when it is the list of exclusions.